atak/ATAK/app/src/main/java/com/atakmap/android/cot/bluetooth/BluetoothSerialCoTManager.java

```diff
@@ -0,0 +1,27 @@
+package com.atakmap.android.cot.bluetooth;
+
+import com.atakmap.android.bluetooth.BluetoothCotManager;
+import com.atakmap.android.maps.MapView;
+
+/**
+ * Manages the actual Bluetooth channel and socket reading for CoT string received over LoRaWAN.
+ */
+public class BluetoothSerialCoTManager extends BluetoothCotManager {
+
+    private static final String TAG = "BluetoothSerialCoTManager";
+
+    private static final byte[] address = new byte[] {
```

Comment:
Recommend that this not publish via a multicast address utilizing a TTL of 1.   Please utilize 
final CotEvent ce = CotEvent.parse(cotString);
CotMapComponent.getInternalDispatcher().dispatch(ce)

Reply:
@bisgroves I will consider this change. The idea of using multicast was based on a use case scenario. Say your are a member of a hunting party. Each android device is connected via a ZeroTier network. The trackers are using RF to communicate position. Once position data is received the receiving device converts the data to CoT and send it over Bluetooth Serial to ATAK and then ATAK repeats the received CoT over multicast. Now each of the connected devices can see the received tracker's information. Does "CotMapComponent.getInternalDispatcher().dispatch(ce)" cause an event that would rebroadcast the received CoT over the network?